### DISCUSSION TITLE
Same study to send multiple AE Title

### DISCUSSION THREAD

**Discussion by esefberfu** [dcm4che/dcm4chee-arc-light#3640] [Q&A] [answered]
My Dcm4chee server has 2 AETitle that is name TESTAE and TESTAE2.
All studies send first to TESTAE. After that some studies must send to TESTAE2.
But if exatcly same study send again to TESTAE2, retrieve aetitle is not changing and this study is showing under TESTAE.
How can i store seperate the study both on TESTAE and TESTAE2

**Comment by gunterze** [comment, marked as answer]
Retrieve AET is stored on Series level on receive of first object of the Series. It does not get updated on receive of the same or other object of the Series by a different Archive AE.

**Comment by esefberfu** [comment]
Thanks for answer. I solved this problem by create attribute coerce for each aetitle. Attribute coerce add suffix to study, series and instance UID according to aetitle
